.travis.yml

```diff
@@ -54,6 +54,16 @@ notifications:
     on_failure: always
 jobs:
   include:
+    - stage: test
+      python: 3.7
+      dist: xenial
+      env:
+        - TOXENV=check
```

Comment:
I think we can remove the `env` block. We don't want to override TOXENV. It should be set by the top-level `before_install`.